docs/source/rbac.rst

```diff
@@ -292,108 +295,197 @@ For example:
 
 .. _rbac-using_rbac:
 
+Automatically granting roles based on the user LDAP group membership
+--------------------------------------------------------------------
+
+.. note::
+
+   This functionality is only available in |st2| v2.3.0 and above when LDAP auth backend is used
+   for authentication.
+
+In addition to manually assigning roles to the users based on the definitions in the
+``/opt/stackstorm/rbac/assignments/`` directory, |st2| also supports automatically granting roles
+to users on authentication based on the LDAP groups membership.
+
+This comes handy in enterprise environments and makes |st2| user provisioning easier and faster.
+It means operator doesn't need to manually write and manage RBAC role assignment files on disk,
```

Comment:
Yeah, we kinda interchangeably use operator / administrator term through the code base and docs.

Would be good to perhaps settle on one term and use it consistently.